fraud-detection-update-sample/main.py

```diff
@@ -17,7 +17,26 @@
 app = initialize_app()
 
 # Fields that are not inputs to the fraud detection, and should not trigger a re-run
-OUTPUT_FIELDS = set(["validity", "validity_details", "water_pct", "land_use_anthropic_pct", "land_use_primary_vegetation_pct", "land_use_secondary_vegetation_or_regrowth_pct", "alerts"])
+OUTPUT_FIELDS = set([
+    "validity",
+    "validity_details",
+    "water_pct",
+    "land_use_anthropic_pct",
+    "land_use_primary_vegetation_pct",
+    "land_use_secondary_vegetation_or_regrowth_pct",
+    "alerts",
+    "reference_oxygen_isoscape_name",
+    "reference_oxygen_isoscape_creation_date",
+    "reference_oxygen_isoscape_precision",
+    "d18O_cel_sample_mean",
+    "d18O_cel_sample_variance",
+    "d18O_cel_reference_mean",
+    "d18O_cel_reference_variance",
+    "validity_details",
+    "validity",
+    "p_value",
+    "p_value_threshold"
```

Comment:
these should really be within validity_details as they are implementation details of the validity.  If we have to have it all flat, so be it, but it'd be nice to group these somehow.